server/src/main/java/org/elasticsearch/index/codec/ES93TSDBZSTDCompressionLucene103Codec.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+package org.elasticsearch.index.codec;
+
+import org.elasticsearch.common.util.BigArrays;
+
+public class ES93TSDBZSTDCompressionLucene103Codec extends TSDBCodecWithSyntheticId {
```

Comment:
Based on my comment in `CodecService`, maybe we just need this codec implementation?

Reply:
Good point. In released versions we endup using zstd based stored fields otherwise endup using default fast compression for default codec. So I think we need to keep both here.